openassessment/xblock/staff_area_mixin.py

```diff
@@ -343,6 +343,10 @@ def get_student_submission_context(self, student_username, submission):
             'student_username': student_username,
         }
 
+        if submission['file_url'] is not None:
```

Comment:
Nit: it seems a shame that every mixin has to know to add the ``file_upload_type``. This seems like a brittle way to have added the feature (I know that wasn't you). I wonder if there's some way to share this logic with a helper method that knows how to add file upload context to a submission.